### ISSUE TITLE
How to determine which step saved model is optimal？

### ISSUE THREAD

**Issue by canghaiyunfan** [open-mmlab/mmdetection#1785]
When finished training with 12 epochs, there are 12 checkpoint foles. The last one may be not best,how can I determine which one is the best model?

**Comment by thomashossler** [comment]
depending what metric you care about. Plot the loss functions, calculate the metrics per epoch and pick the best one :)
